types/which-polygon/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for which-polygon 2.2
+// Project: https://github.com/mapbox/which-polygon
+// Definitions by: Kyle Hensel <https://github.com/k-yle>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.0
```

Comment:
is this really worth of adding, in context of public 3rd party lib enforcing this TS version?

Reply:
Hi @peterblazejewicz, we need `≥4.0` so that we can use named tuple elements, which is a major benefit for this package.

For example:
```ts
type BBox = [minLat: number, maxLat: number, minLng: number, maxLng: number]
```

is _much_ more useful than:
```ts
type BBox = [number, number, number, number]
```

This is especially helpful since different libraries use a different order for `BBox`, which is a common source of confusion.

Reply:
thx! Just wanted to be sure, we are on same track, thx!